src/components/TableHeaderCell/TableHeaderCell.tsx

```diff
@@ -65,12 +71,27 @@ export const TableHeaderCell = ({
   const componentClassName = clsx(
     styles['table__header-cell'],
     className,
-    sortDirection === 'ascending' && styles['table-header-cell--ascending'],
-    sortDirection === 'descending' && styles['table-header-cell--descending'],
-    !sortDirection && styles['table-header-cell--none'],
-    sortDirection && styles['table-header-cell--ascending'],
+    sortable &&
+      (sortDirection === 'ascending' && styles['table__header-cell--ascending'],
+      sortDirection === 'descending' &&
+        styles['table__header-cell--descending'],
+      sortDirection && styles['table__header-cell--ascending']),
   );
 
+  const iconName =
+    sortDirection === 'ascending'
+      ? 'arrow-narrow-up'
+      : sortDirection === 'descending'
+      ? 'arrow-narrow-down'
+      : 'unfold-more';
```

Comment:
I thought we decided against using this "unfold-more" icon for sortable table headers. Worth bringing up with the team

Reply:
I couldn't find anything in Figma discussing that. It can always be swapped out with a change order after deployment.

Reply:
Also something I just noticed: when you click on the sortable table header and the icon cycles between up, down, and unfold more, it actually looks pretty tight, i.e. all 3 icons are using the same visible space

Reply:
Tight meaning "good" or tight meaning too close together? Let's tighten up styles by adding some space in between the icon and text as well as centering that icon.

Reply:
Tight as in good. Tight, tight tight!

![Tight!](https://s3.scoopwhoop.com/raj/1004/tuco.gif)